src/content/documentation/publish/distribute-manifest-versions.md

```diff
@@ -0,0 +1,110 @@
+
+So, if your user base is large, or you have requirements to provide support for older Firefox versions for an extended period, we recommend that you consider using this alternate distribution for your MV3 version.
+
+If you choose to remain on MV2 and wait to transition to MV3 later, there are steps you can take in MV2 that [move your extension closer to what is necessary in MV3](https://blog.mozilla.org/addons/2022/10/31/begin-your-mv3-migration-by-implementing-new-features-today/). As usual, if your extension depends on features only available to recent Firefox versions, specify the compatible Firefox version in the [`strict_min_version`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_specific_settings) field in manifest.json. Older Firefox extensions will not receive an update, and a new user on an old Firefox version can install an older version of the extension using the “See all versions” link at the extension listing on AMO.
+
+{% endcapture %}
+{% include modules/column-w-toc.liquid,
+  id: "transfer-ownership"
+  content: content_with_toc
+%}
+
+<!-- END: Content with Table of Contents -->
+<!-- Single Column Body Module -->
+
+{% capture content %}
+
+## How to sign an MV3 version
+
+To get a signed version of your MV3 extension:
+
+1. On AMO, when you upload the MV3 extension version, change “Where to host version”  to “on your own”.
+   ::: note
+   You need to change it back to “on this site” for MV2 versions, as the most recently used channel is remembered.
+   :::
+2. For API uploads, specify `channel=unlisted` for the self-distribution channel.
+
+See [Self distribution](/documentation/publish/submitting-an-add-on/#self-distribution), for more details on the process.
+
+{% endcapture %}
+{% include modules/one-column.liquid,
+  id: "listing-on-amo"
+  content: content
+%}
+
+<!-- END: Single Column Body Module -->
+<!-- Single Column Body Module -->
+
+{% capture content %}
+
+## How to distribute the MV3 version
+
+Self-distributed versions cannot be downloaded by users directly from AMO. You must distribute the experimental XPI to users yourself, such as by a download from your website, email, etc.
+
+Remember that:
+* When you distribute the XPI file from a website that is not AMO, you also need to add the `install_origins` key to the manifest. This key lists the origin of the host that you place the XPI on.
+* To support automatic updates to newer MV3 versions an [update manifest file](/documentation/manage/updating-your-extension/) must be created and made available, and the url to this update manifest [specified in the extension’s manifest.json](/documentation/manage/updating-your-extension/).
+
+See [Distributing an add-on yourself](/documentation/publish/self-distribution/), for more details on the process.
```

Comment:
Another question on sentence case for a link, compared to other links that use lower case..

Reply:
Because it's the page title, link text in other cases isn't (or shouldn't be) to page titles. (I'll check.)